Develop  lab1

vladyslavmyronchuk opened the pull request.

vladyslavmyronchuk committed:
Create main.cpp with a basic Hello World program

vladyslavmyronchuk committed:
Merge pull request #1 from vladyslavmyronchuk/feature/develop/01-HelloWorld

Create main.cpp with a basic Hello World program

vladyslavmyronchuk committed:
Додано базовий CMakeLists.txt

vladyslavmyronchuk committed:
Changed comment to English in CMakeLists.txt

vladyslavmyronchuk committed:
Merge pull request #2 from vladyslavmyronchuk/feature/develop/02-cmakelist

Added CMakeLists.txt to the branch

vladyslavmyronchuk committed:
Add header for math_operations module with add function

vladyslavmyronchuk committed:
Implement add function in math_operations module

vladyslavmyronchuk committed:
Add unit tests for add function using Google Test

vladyslavmyronchuk committed:
Add Google Test support and test executable setup to CMakeLists.txt

vladyslavmyronchuk committed:
Update CMakeLists.txt for project configuration

vladyslavmyronchuk committed:
Merge branch 'develop' into feature/develop/03-UnitTesting

vladyslavmyronchuk committed:
Merge pull request #3 from vladyslavmyronchuk/feature/develop/03-UnitTesting

Add unit testing

vladyslavmyronchuk committed:
Update ci.cmd with new CI pipeline script

vladyslavmyronchuk committed:
Add ci.sh script for CI pipeline

vladyslavmyronchuk committed:
Merge pull request #4 from vladyslavmyronchuk/feature/develop/04-unit_testing

Add  CI-scripts

vladyslavmyronchuk committed:
Create ci.yml

vladyslavmyronchuk committed:
Add CI

This CI workflow builds and tests the project on multiple operating systems using CMake.

vladyslavmyronchuk committed:
Refactor CI workflow for clarity and updates

vladyslavmyronchuk committed:
Merge pull request #5 from vladyslavmyronchuk/feature/develop/05-yml_files

Add yml files

PavloFedakNULP reviewed (changes requested):
Please include the history of communication with the AI assistant.

vladyslavmyronchuk committed:
Create chat_history.txt with chat log from this chat

vladyslavmyronchuk committed:
Add chat history log to chat_history.txt

Recorded the entire chat history in a text file as requested, including both user and Copilot responses.

PavloFedakNULP reviewed (changes requested):
Please include the history of your requests and responses with the AI assistant.

vladyslavmyronchuk commented:
> Please include the history of your requests and responses with the AI assistant.

Я добавив всю історію запитів і відповідей в один текстовий файл chat_history.txt,перегляньте будь ласка
